user/build-stages/share-files-s3.md

```diff
@@ -11,14 +11,17 @@ This example has 2 build stages:
 Here's what the `.travis.yml` config could look like:
 
 ```yaml
+dist: xenial
+
 env:
   global:
     # include $HOME/.local/bin for `aws`
     - PATH=$HOME/.local/bin:$PATH
 
 before_install:
-  # set up awscli packages
-  - pip install --user awscli
+  - pyenv global 3.7.1
+  - pip install -U pip
```

Comment:
It would be better to more explicitly use __--upgrade__ than __-U__ to avoid confusion because pip also has a __--user__ option.